__tests__/app.test.js

```diff
@@ -86,3 +87,22 @@ describe("GET /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("should return 200, an array of objects with username name and avatar_url properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ users: body }) => {
+        users.forEach((users) => {
```

Comment:
remember to check the length of the array before using a forEach in a test, otherwise you can get false positives back